libmscore/measure.cpp

```diff
@@ -4222,5 +4222,14 @@ void Measure::computeMinWidth()
       computeMinWidth(s, x, isSystemHeader);
       }
 
+int Measure::calculateAdjustedNo()
```

Comment:
I think this possibly should be a member of `MeasureBase` instead, seeing as that's the class in which the offset is stored. Not a massive problem, but I think it would make slightly more sense in the current object hierarchy.

Reply:
Thank you, I agree with that. I've fixed that now.